package.json

```diff
@@ -11,12 +11,15 @@
     "node": ">=10.0"
   },
   "dependencies": {
+    "@babel/helper-create-regexp-features-plugin": "^7.8.3",
     "@bugsnag/js": "^6.2.0",
     "@bugsnag/plugin-react": "^6.2.0",
+    "@material-ui/core": "^4.9.0",
```

Comment:
I love notistack, and I think it already has the dependency to material-ui, we shouldn't need it as far as I remember. Also I know it is quite large, so maybe we can import only the components we need from that library...what do you think?

Reply:
I thought so too, but it complained! Anyway, I'm closing this PR because I agree that we should add notistack to Portal directly …